### PR TITLE
Wwise file support

### PR DESCRIPTION
**Pull request by ADawesomeguy**
Implemented:
- [x] Command for converting WEM to OGG in CLI

GUI is too complicated for me so I'm gonna push this and someone else can add it to the GUI if needed.

### CODE REVIEW

File: WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                outpath = Path.ChangeExtension(path, ".ogg");
+            }
+
+            if (wem)
+            {
+                var file = File.ReadAllBytes(path);
+                using var ms = new MemoryStream(file);
+                using var br = new BinaryReader(ms);
+
+                var inBuffer = br.ReadBytes(file.Length);
```
Comment (rfuzzo) [inline WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs:28]: As far as I can see you just need `var inbuffer = File.ReadAllBytes(path);` here. No need for streams if you just use the byte array in the end right?

File: WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using WolvenKit.Core.Wwise;
+
+namespace CP77Tools.Tasks
+{
+    public partial class ConsoleFunctions
+    {
+        #region Methods
```
Comment (rfuzzo) [inline WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs:8]: No need for regions if it's just a small file and has no other regions.

File: WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+                File.WriteAllBytes(outpath, oggBuffer);
+
+                _loggerService.Success($"Finished converting {path} to OGG: {outpath}");
+            }
+
+            return 1;
+        }
+
+        #endregion Methods
```
Comment (rfuzzo) [inline WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs:45]: No need for regions if it's just a small file and has no other regions.

**Review by rfuzzo — CHANGES_REQUESTED** [review]
Looks good, just the comment about the streams :)

File: WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs (hunk 1 above)
Comment (ADawesomeguy) [reply WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs:28]: Ah yeah sorry, I had that in my test but for this file I'd copied the Kraken stuff. I'll fix it ASAP.

**Review by rfuzzo — APPROVED** [review]
Lgtm

**Pull request merged**